ano/environment.py

```diff
@@ -74,7 +74,9 @@ class Environment(dict):
     arduino_user_dir_guesses = [
         'libraries',
         os.path.expanduser("~/Documents/Arduino"),
-        os.path.expanduser("~/Arduino")
+        os.path.expanduser("~/Arduino"),
+        os.path.expanduser("~/.arduino15"),
+        os.path.expanduser("~/.arduino")
```

Comment:
This is the first part of the significant change.
There may be other versions than .arduino15 but I can't confirm.